server_admin/topics/realms/proc-using-admin-console.adoc

```diff
@@ -27,4 +27,4 @@ image:{project_images}/admin-console.png[Admin Console]
 +
 * Click the top right list to view your account or log out.
 +
-* Hover over a question mark *?* icon to show a tooltip text that describes that field. The image above shows the tooltip in action.
+* Click over a question mark *?* icon to show a tooltip text that describes that field. The image above shows the tooltip in action.
```

Comment:
This should actaully be Click not Click over. Also it should in conditional text for the new Admin Console only. The hover line should remain but be for the old Admin Console.

Reply:
Thanks for feedback, all done @andymunro

Reply:
You have zero xxxx-prod.adoc files unless you include those file somewhere.